WTWSMS/common/religions/00_religions.txt

```diff
@@ -1527,6 +1540,7 @@ muslim = {
 	}
 	druze = {
 		graphical_culture = muslimgfx
+		has_coa_on_barony_only = yes
```

Comment:
I use "kebab" as a term for all Muslims, and this is not the Paradox Forums. Hence, I will not use that term there. I'm banned anyway.

What I mean is that the Druze are not mainstream and would not use mainstream Islamic symbols when they have their own.